includes/virtual-machines-common-classic-resource-manager-migration-overview.md

```diff
@@ -87,8 +87,6 @@ The following screenshots show how to upgrade a Classic storage account to an Az
 ### Migration of unattached resources
 Storage Accounts with no associated disks or Virtual Machines data may be migrated independently.
 
-Network Security Groups, Route Tables & Reserved IPs that are not attached to any Virtual Machines and Virtual Networks can also be migrated independently.
-
```

Comment:
Is this no longer supported for migration?

Reply:
@tanmaygore are you asking me? I wrote in my comment the reason that I removed that - it doesn't seem correct from the Deep Dive doc info.

Reply:
Confirmed with the developers. Migration of NSGs, Route Tables & Reserved IP is possible even when they are not attached. 

The other document is incorrect. 
@dcbrown16 , @ktoliver can you please update the docs. Thanks!